### PR TITLE
Add battery status message for v1

### PR DESCRIPTION
**Pull request by Katawann**
Following the new messages in MAVSDK-Proto, it is required to implement the logic and conversion if necessary of the battery status. New messages are:
   - temperature_degc
   - current_battery_a
   - capacity_consumed_ah

Depends on https://github.com/mavlink/MAVSDK-Proto/pull/297.

### CODE REVIEW

**Comment by Katawann** [comment]
Hi @julianoes and @JonasVautherin 

ready for a review :) For the proto check, I guess it requires to update the proto submodule but we need to merge in MAVSDK proto first right ?

File: src/integration_tests/telemetry_async.cpp [hunk 1]
```diff
@@ -341,7 +341,10 @@ void print_gps_info(Telemetry::GpsInfo gps_info)
 void print_battery(Telemetry::Battery battery)
 {
     std::cout << "Battery " << battery.id << ": " << battery.voltage_v << " v,"
-              << "remaining: " << int(battery.remaining_percent * 1e2f) << " %" << '\n';
+              << "temparutre: " << battery.temperature_degc << " degC,"
```
Comment (JonasVautherin) [inline src/integration_tests/telemetry_async.cpp:344]:
```suggestion
              << "temperature: " << battery.temperature_degc << " degC,"
```

**Review by JonasVautherin — APPROVED** [review]
LGTM

**Review by JonasVautherin — CHANGES_REQUESTED** [review]
Well actually just the typo to fix :innocent:

**Comment by JonasVautherin** [comment]
Does it request an update of `mavsdk-proto`? I see the proto check fails

**Comment by Katawann** [comment]
> Does it request an update of `mavsdk-proto`? I see the proto check fails

Yes but should it be merged first so than I can update it ? https://github.com/mavlink/MAVSDK-Proto/pull/297

**Comment by julianoes** [comment]
Thanks @Katawann and sorry it took so long.

**Comment by Katawann** [comment]
Any plan for a big release soon (including this feature) ? Otherwise I will build this modification on top of v1.4.10 next week

**Comment by julianoes** [comment]
I'm hoping for end of Feb for v2.

**Comment by Katawann** [comment]
Ok cool ! Thanks for the answer. Looking forward to it :)

**Pull request merged**